docs/install/index.md

```diff
@@ -32,6 +32,10 @@ pip install onnxruntime
 pip install onnxruntime-gpu
 ```
 
+```bash
+pip install onnxruntime-cann
```

Comment:
these instructions are for installing onnxruntime packages available on pypi.
onnxruntime-cann isn't a published pypi package right? Please remove these lines.

Reply:
We considered to publish onnxruntime-cann based on master branch of onnxruntime on pypi before. However, we now decided to publish it with version 1.13.0 of onnxruntime.

These lines have been removed and will be updated when 1.13.0 version are released